app/api/models/statistics.py

```diff
@@ -12,11 +13,11 @@
 logging.getLogger().handlers = [InterceptHandler()]
 
 class Homology(BaseModel):
-    coverage: float = Field(
+    coverage: float | None = Field(
         alias=AliasChoices("compara.homology_coverage", "compara.homology_coverage "),
         default=None,
     )
-    reference_species_name: str = Field(
+    reference_species_name: str | None = Field(
         alias=AliasChoices(
             "compara.homology_reference_species", "compara.homology_reference_species "
```

Comment:
suggestions: remove space at the end of `"compara.homology_reference_species "` (also on line 17), add `default=None`

Reply:
@veidenberg 
I don't know but it seem to be working.
default=None:
https://github.com/Ensembl/ensembl-web-metadata-api/blob/cca18c331147ca18a2f4197824c963710b78ceae/app/api/models/statistics.py#L24

Reply:
The space sems to be a "temporary" hack until fixed upstream (see the comment [here](https://github.com/Ensembl/ensembl-web-metadata-api/pull/35#discussion_r1403144366))

Reply:
@Mehrnaz-Charkhchi do you want to add it to this PR or shall I open a new one?
```python
Field(alias="compara.homology_reference_species ", default=None)
```
(same for `compara.homology_coverage`)